app/assets/javascripts/client/inventories/analysis_dashboard.html

```diff
@@ -0,0 +1,70 @@
+<div class="dashboard">
+  <div class="row row-bg row-header">
+    <div class="col-md-12">
+      <h1 class="greeting">{{analysisDashboard.inventory.name}}</h1>
+      <h4 class="byline">organized by: <a href="#">{{analysisDashboard.inventory.facilitator.full_name}} </a>
+        on {{analysisDashboard.inventory.created_at | amDateFormat:'MMMM Do, YYYY'}}
+      </h4>
+    </div>
+  </div>
+  <div class="row">
+    <div id="participants" ng-if="analysisDashboard.currentAnalysis.is_facilitator">
```

Comment:
This facilitator check will _hide_ the participants list as opposed to showing it.  I believe an earlier version of the inventory dashboard did something similar but it has [since been corrected](https://github.com/MobilityLabs/pdredesign-server/blob/199-analysis-dashboard-page/app/assets/javascripts/client/inventories/dashboard.html#L23).  The if-check needs to be moved to the div containing the invitation control pieces.